draft-irtf-cfrg-opaque.md

```diff
@@ -431,6 +429,7 @@ public key.
 material for the AKE from the Envelope. This function raises an error if the
 private seed cannot be used for recovering authentication material from the
 input envelope.
+- Ne: The size of the envelope, which may vary between mechanisms.
```

Comment:
```suggestion

The size of the envelope is a constant denoted as `Ne`. Its size may vary between key recovery methods.
```

Reply:
I used the convention in the previous section where constants are explicitly itemized. I suggest this change because finding this parameter in the middle of a sentence is more difficult to parse.

Reply:
Well, it's not a function, so the sentence introducing the list would need to change if we were to take this suggestion.

Reply:
updated the sentence.

Reply:
Looks good!